src/pages/AppDetails/AppDetailsPage.tsx

```diff
@@ -164,22 +166,15 @@ class AppDetails extends React.Component<AppDetailsProps, AppDetailsState> {
     // Conditional Traces tab
     if (this.props.jaegerInfo && this.props.jaegerInfo.enabled) {
       if (this.props.jaegerInfo.integration) {
-        const jaegerTitle =
-          this.state.nbErrorTraces > 0 ? (
-            <>
-              Traces <ExclamationCircleIcon color={PfColors.Red200} />{' '}
-            </>
-          ) : (
-            'Traces'
-          );
         tabsArray.push(
-          <Tab eventKey={4} style={{ textAlign: 'center' }} title={jaegerTitle} key={tracesTabName}>
+          <Tab eventKey={4} style={{ textAlign: 'center' }} title={'Traces'} key={tracesTabName}>
             <TracesComponent
               namespace={this.props.match.params.namespace}
               target={this.props.match.params.app}
               targetKind={'app'}
-              showErrors={this.state.nbErrorTraces > 0}
+              showErrors={false}
```

Comment:
This is a functional change, we won't restrict the traces view to show only errors where there are some, is that the intent?
I'm +1 for this change anyway.
If so, I guess this `showErrors` props isn't used anymore and could go away.

Reply:
Yes, sorry, I should leave a note.
Also, this was not consistency as error icons only happen on "app/traces" details tab, not on "workload/traces" nor "service/traces".

Also, with all new logic, I guess it was good to reconsider that icon, because in the same way, it should appear also in the metrics and log tabs (as error would happen there).

So, yes, I guess that for this moment, we can remove that icon in the tab, and delegate in the filter options used on every tab.